app/javascript/tests/mocks/mockPatients.js

```diff
@@ -526,6 +524,7 @@ const mockPatient2 = {
   black_or_african_american: null,
   case_status: 'Confirmed',
   closed_at: '2020-09-13T20:32:02.000Z',
+  common_exposure_cohorts: [mockCommonExposureCohort1, mockCommonExposureCohort2],
```

Comment:
is this needed on the mock patient? I wouldn't think so based on how they are passed in as props

Reply:
Ah I see it's needed in the Patient component.  I'd prefer to append them to the mock patient in that object manually than do it here, since the cohorts don't exist on the patient model.  I can do this if you want

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/913b2d76b4971c4f97c842472610dff60f97b69c